web-api/src/net.rs

```diff
@@ -125,7 +125,7 @@ where
     // `spawn` it on the tokio runtime. This hands off the responsibility to poll the server to
     // tokio. At the same time we keep the `JoinHandle` so that we can wait for the server to
     // stop and get it's return value (we don't have to await `term_handle`).
-    let span = info_span!(parent: None, "web-api");
+    let span = error_span!(parent: None, "web-api");
     span.follows_from(Span::current());
```

Comment:
:pushpin:  while this the correct way to do it, with how we currently log things this won't get logged in any usefull way :see_no_evil: 

This means we e.g. can't look up the test_id of the "current" span when logging event in the server.